docs/helm/README.md

```diff
@@ -57,6 +53,38 @@
     **Content:** `{ error : "error message" }`
 
 
+
+**Get Helm Release History**
+----
+  _Returns installed helm release history in JSON_
+
+* **URL**
+
+     `/api/helm/release/history`
+
+* **Method:**
+  
+  `GET` 
+  
+*  **URL Params**
+ 
+   `ns=[string]` - Namespace
+   
+   `name=[string]` - Helm Release Name
```

Comment:
URL Param for release name is not consistent across all API endpoints. `/api/helm/release` endpoint needs URL param as `release_name`. Can you also update that endpoint to be consistent?

Reply:
We would like to create a separate PR that could go into the code quicker. @akashshinde

Reply:
@rohitkrai03 the idea is to use `name` instead of `release_name`, that is the reason you would see `name` param in all endpoints implemented in this PR. Going forward we will be renaming other endpoints url param from `release_name` to `name` in separate PR.